src/Behat/Behat/Output/Node/EventListener/JUnit/JUnitOutlineStoreListener.php

```diff
@@ -53,7 +54,18 @@ public function __construct(SuitePrinter $suitePrinter)
      */
     public function listenEvent(Formatter $formatter, Event $event, $eventName)
     {
-        $this->captureOutlineOnBeforeOutlineEvent($event);
+      $this->captureOutlineOnBeforeOutlineEvent($event);
+      if ($event instanceof SuiteTested) {
+        foreach ($event->getSpecificationIterator() as $node) {
+          if ($node instanceof FeatureNode) {
```

Comment:
I personally prefer to `continue` if not `instanceof`. That helps readability by reducing the depth of indentation.